app.py

```diff
@@ -31,8 +31,9 @@ def about():
 
 @app.get('/search')
 def search():
+    q = request.args.get('q', '')
     current_page = "search"
-    return render_template('search.html') 
+    return render_template('search.html', search_query=q) 
```

Comment:
if this makes it a requirement to have a query every time you look for the search page, consider placing this in an if statement to prevent making this a breaking change of other places referencing search

`if (q exists)
    return render_template('search.html', search_query=q)
else
    return render_template('search.html')`
    
may not be necessary, dont have time to look rn, just mentioning it in case this breaks something in other locations calling search.html without a q value

Reply:
it doesn't, that only is filled if the search bar form is filled, it also can be blank and the normal search page comes up. #tried&tested